src/xrpld/app/tx/detail/VaultClawback.cpp

```diff
@@ -158,8 +208,15 @@ VaultClawback::assetsToClawback(
     AccountID const& holder,
     STAmount const& clawbackAmount)
 {
+    if (clawbackAmount.asset() != vault->at(sfAsset))
+    {
+        // preclaim should have blocked this case, this is now an internal error
+        JLOG(j_.error()) << "VaultClawback: asset mismatch in clawback.";
+        return Unexpected(tecINTERNAL);  // LCOV_EXCL_LINE
+    }
```

Comment:
With a log and error return, you should use `LCOV_EXCL_START` and `LCOV_EXCL_STOP`.
```suggestion
    if (clawbackAmount.asset() != vault->at(sfAsset))
    {
        // LCOV_EXCL_START
        // preclaim should have blocked this case, this is now an internal error
        JLOG(j_.error()) << "VaultClawback: asset mismatch in clawback.";
        return Unexpected(tecINTERNAL);
        // LCOV_EXCL_STOP
    }
```

Reply:
Done.